pypd/models/incident.py

```diff
@@ -100,6 +100,116 @@ def reassign(self, from_email, user_ids):
                               data=data,)
         return result
 
+    def add_responders(self, from_email, requester_id, message, user_ids=None, escalation_policy_ids=None):
```

Comment:
If someone uses this function and only has an `escalation_policy_id`, I believe this function would assume that argument is a `user_id` and then `escalation_policy_ids` would be set to `None`. 

For example: `add_responders("from@email.com", "requester_id", "Please join call", "ep_id")` no matter what would result in the `escalation_policy_ids` argument being `None` even if `ep_id` is intended to be the EP ID. 

I wonder if it would be better to just take in a `requested_responder_id` (or something like that) and add a field that specifies what type of responder it is (`escalation_policy_id`  or `user_id`).

Reply:
I did just realize you're likely making `user_ids` and `escalation_policy_ids` plural for cases where a user might want to add multiple responders via this API call. You could consider removing the default so the user must at the very least add an empty string or array to that argument.